styles/godoc.less

```diff
@@ -25,24 +19,38 @@
   &::before {
     content: "";
     position: absolute;
-    top: -4px;
-    left: 0;
-    border: 4px solid;
-    border-color: transparent transparent @godoc-bg @godoc-bg;
+    top: -@godoc-border-width * 3;
+    left: -@godoc-border-width;
+    border: @godoc-border-width * 1.5 solid;
+    border-color: transparent transparent @overlay-border-color @overlay-border-color;
+  }
+
+  .close {
+    position: absolute;
+    top: @component-padding/2;
+    right: @component-padding/2;
+    color: lighten(@text-color, 50%);
+    text-shadow: none;
+    &::before {
+      margin-right: 0;
+    }
   }
 }
 
-.close {
-  color: lighten(@text-color, 50%);
-  text-shadow: none;
+#godoc-message {
+  // overflow: auto; // TODO: still not getting scroll bars...
```

Comment:
Yeah, ideally it would try to figure out how much space is left at the bottom, if there isn't much, it would show the overlay above. And maybe even taking scroll position into account. I think `autocomplete-plus` tries to do that, but that's probably custom made.